packages/firebase_remote_config/lib/src/remote_config_fetch_throttled_exception.dart

```diff
@@ -0,0 +1,23 @@
+part of firebase_remote_config;
+
+class FetchThrottledException implements Exception {
+  DateTime _throttleEnd;
+
+  FetchThrottledException._({int endTimeInMills = 43200}) {
+    _throttleEnd = new DateTime.fromMillisecondsSinceEpoch(endTimeInMills);
+  }
+
+  DateTime get throttleEnd => _throttleEnd;
+  String get msg {
```

Comment:
This getter seems to be unused and can be removed

Reply:
Done